src/ui/layout/AppLayout.vue

```diff
@@ -85,7 +85,8 @@
         <template #controls>
           <button
             class="c-icon-button l-shell__reset-tree-button icon-folders-collapse"
-            title="Collapse all tree items"
+            aria-label="Expand or Collapse all tree items"
+            title="Expand or Collapse all tree items"
```

Comment:
@ozyx is this supposed to actually work on openmct-yamcs? Clicking this element doesn't expand anything in the tree

Reply:
It only collapses the tree